docs/getting-started/agentless-troubleshooting.asciidoc

```diff
@@ -0,0 +1,47 @@
+[[agentless-integration-troubleshooting]]
+= Agentless integrations FAQ
+
+Frequently asked questions and troubleshooting steps for {elastic-sec}'s agentless CSPM integration.
+
+[discrete]
+== When I make a new integration, how long until the agent appears on the Integration Policies page?
+
+After you create a new agentless integration, the new integration policy may show a button that says **Add agent** instead of the associated agent for several minutes during agent enrollment. No action is needed other than refreshing the page once enrollment is complete.
+
+[discrete]
+== How do I troubleshoot an `Offline` agent?
+
+For agentless integrations to successfully connect to {elastic-sec}, your Fleet server host value must be the default. Otherwise, the agent status that appears on the Fleet page will be `Offline`, and logs will include the error `[elastic_agent][error] Cannot checkin in with fleet-server, retrying`. 
```

Comment:
```suggestion
For agentless integrations to successfully connect to {elastic-sec}, the {fleet} server host value must be the default. Otherwise, the agent status on the {fleet} page will be `Offline`, and logs will include the error `[elastic_agent][error] Cannot checkin in with fleet-server, retrying`. 
```